hypervisor/src/vm.rs

```diff
@@ -297,7 +297,12 @@ pub trait Vm: Send + Sync {
         userspace_addr: u64,
     ) -> Result<()>;
     /// Get dirty pages bitmap
-    fn get_dirty_log(&self, slot: u32, memory_size: u64) -> Result<Vec<u64>>;
+    fn get_dirty_log(
+        &self,
+        slot: u32,
+        #[cfg(feature = "mshv")] base_gpa: u64,
```

Comment:
As we now only need one new argument `gpa` for MSHV (and it is not MSHV-specific information), I think we can drop the feature gate here (like what we did with the argument `slot`). It makes it easier to support KVM and MSHV in a single binary in the future. WDYT?